Sources/TuistGenerator/Mappers/ResourcesProjectMapper.swift

```diff
@@ -78,10 +85,9 @@ public class ResourcesProjectMapper: ProjectMapping { // swiftlint:disable:this
             sideEffects.append(sideEffect)
         }
 
-        if project.isExternal,
```

Comment:
Please note I removed the external flag, I think we should comply to SPM and generate those if any objc source files are in + resources...